k8s/helm-charts/seldon-core-v2-setup/templates/_components-statefulsets.tpl

```diff
@@ -1316,10 +1316,20 @@ metadata:
 spec:
   podSpec:
     containers:
-    - env:
+    - args:
+      - rcd
+      - --rc-no-auth
+      - --config=/rclone/rclone.conf
+      - --rc-addr=0.0.0.0:5572
+      - --max-buffer-memory=$(MAX_BUFFER_MEMORY)
+      command:
+      - rclone
+      env:
       - name: RCLONE_LOG_LEVEL
         value: '{{ hasKey .Values.serverConfig.rclone "logLevel" | ternary .Values.serverConfig.rclone.logLevel
           .Values.logging.logLevel | upper }}'
+      - name: MAX_BUFFER_MEMORY
+        value: 16M
```

Comment:
Can we increase this up a bit? I.e 64M? Just because 16M will mean files will be downloaded sequentially (only space for 1 concurrent transfer before exceeding the max buffer memory).